content/posts/blog/easy-xmpp-challenge.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Easy XMPP: The Challenges
+date: 2017-08-08
+author: ge0rg
+blog_id: blog
+category: Easy XMPP
+---
+
+Over the last years, the XMPP community has had a hard time competing with
+other Instant Messaging implementations, especially in the mobile / smartphone
+ecosystems. By focusing a small part of our resources on user experience (UX),
+we can gain significant improvements. This article is the first in a series of
+"Easy XMPP" posts: easy ways for application developers to make XMPP easy to
+use.
+
+## Complexity of Federation
+
+As opposed to most other Instant Messaging solutions, XMPP is a federated
+protocol. That allows everyone to run their own servers, at the cost of
+additional complexity for users:
+
+* the user identifier always consists of a *user* and a *domain* part,
+* there is no central registry that will consume your phone book and tell you
+  who else is using XMPP,
+* some servers might be running an older stack not supporting modern features,
+  etc.
+
+This inherent complexity, together with many developers' lack of attention to
+good UX, have left us in a situation where on-boarding of new users and
+finding contacts is painfully hard, especially when compared to
+proprietary/centralized IM solutions.
+
+We can not remove the inherent complexity of federation without replacing XMPP
+with a completely different protocol. However, there is another federated
+system that is well established and used by people all over the world:
+**email**. XMPP and email share the same basic principles - they are
+federated, user identifiers are *user@domain* and it is possible (albeit less
+common) to run your own server (or to get your own domain hosted with a
+commercial provider).
+
+We can leverage users' knowledge about how email works, learn from over forty
+years of email evolution, and apply ideas from modern UX design on top, to
+make XMPP easier to use today.
+
+## Our Challenges
+
+There are several areas where we as a community need to improve. This post
+focuses on terminology and the first-time user experience. Topics that will be
+covered by subsequent posts are contact discovery, group chats, handling of
+spam, and more.
+
+### Terminology
+
+The XMPP terminology is driven by technical requirements and exposes
+complexities of the protocol. Normal IM users don't want to know about *PubSub
+Publish Options*, *asymmetric presence subscription* in their *roster*, or
+*MUC-PM Carbons*. All they care about is that their messages can
+be read by the respective receiver.
+
+It is our task as the developer community to create understandable
+abstractions of the technical complexities, and to set up a
+[common glossary](https://wiki.xmpp.org/web/Usability/Glossary)
```

Comment:
Technical nitpits which are not related to the content in general: I wonder whether it would make (more) sense to have this as Informational XEP. Benefits:

1. can be included in compliance suites
2. better discoverability for new developers